pkg/http/subscriber.go

```diff
@@ -104,7 +104,8 @@ func (s *Subscriber) Subscribe(ctx context.Context, url string) (<-chan *message
 	messages := make(chan *message.Message)
 
 	s.outputChannelsLock.Lock()
-	s.outputChannels = append(s.outputChannels, messages)
+	s.outputChannels[url] = append(s.outputChannels[url], messages)
+	channels := s.outputChannels[url]
```

Comment:
I'm not sure if it's fine to call `s.config.Router.Post` below for each Subscribe separately. Perhaps it should be cached in some way.